the-collab-lab/tcl-35-smart-shopping-list
# Style Smart Shopping App

## Description

This PR added the style for the app to look professional, welcoming, and accessible for users.



## Related Issue

Closes #13 



## Acceptance Criteria

As a user, I want the app to look professional and welcoming.
- Design
- Use vanilla or CSS frameworks.
- Accessibility.



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓   | :link: Update dependencies |
|    | :scroll: Docs              |


## Updates

### Before

![previous-homepage](https://user-images.githubusercontent.com/45172775/143479023-5a4a8248-1862-4a41-979b-d827da211343.jpg)



### After

#### Desktop
![desktop](https://user-images.githubusercontent.com/45172775/143479267-af1f4336-edd5-419f-915d-41dd261c7f05.jpg)

#### Mobile
![mobile](https://user-images.githubusercontent.com/45172775/143479295-39a3430b-716d-40f9-bdb4-d2125883339b.jpg)




## Testing Steps / QA Criteria

- Pull the branch to your machine.
- Run `npm install` to install all dependencies.
- Run `npm start` to start the project and see the style on the browser `localhost:3000`.
- Drag the browser or use the browser's devtools by toggling the device toolbar to see the responsiveness.
- Use WAVE, or axe DevTools, or Lighthouse and screen readers to check on the accessibility.
